Client/Frontend/AuthenticationManager/AuthenticationSettingsViewController.swift

```diff
@@ -157,7 +157,7 @@ class TouchIDSetting: Setting {
         let accessoryContainer = UIView(frame: control.frame)
         accessoryContainer.addSubview(control)
 
-        let gesture = UITapGestureRecognizer(target: self, action: "switchTapped")
```

Comment:
Sorry I commented on the previous line, not the changed line. More of a question about the Selectors in general.